addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -284,8 +292,8 @@ private boolean isHexCode(String code) {
      */
     private String convertHexToGC(String hexCode) throws HexCodeConversionException {
         // Magic number for converting frequency to GC format
-        final int FREQ_CONVERSION_FACTOR = 4145146;
```

Comment:
The cost of initialization seems insignificant when compared to the rest of the method.